tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -44,40 +45,35 @@ class RNTupleModel;
 // clang-format on
 class RFieldDescriptor {
    friend class RNTupleDescriptorBuilder;
+   friend class RFieldDescriptorBuilder;
 
 private:
    DescriptorId_t fFieldId = kInvalidDescriptorId;
    /// The version of the C++-type-to-column translation mechanics
-   RNTupleVersion fFieldVersion;
+   RNTupleVersion fFieldVersion = RNTupleVersion();
    /// The version of the C++ type itself
-   RNTupleVersion fTypeVersion;
+   RNTupleVersion fTypeVersion = RNTupleVersion();
    /// The leaf name, not including parent fields
-   std::string fFieldName;
+   std::string fFieldName = "";
```

Comment:
The `std::string` is anyway default constructed to the empty string

Reply:
Thank you, done in 9e8f0f2e78d8cb1883a873e3b8aeea731517cca6